synapse/config/registration.py

```diff
@@ -54,6 +55,11 @@ def default_config(self, **kwargs):
         # secret, even if registration is otherwise disabled.
         registration_shared_secret: "%(registration_shared_secret)s"
 
+        # Sets the expiry for the short term user creation in
+        # milliseconds. For instance the bellow duration is two weeks
+        # in milliseconds.
+        user_creation_max_duration: 1209600000
```

Comment:
Might be worth calling the `short_term_user_max_duration`, just to spell out it only affects short term users?